Lab 9.5: use --namespace always with the special quota namespace

splattner opened the pull request.

splattner commented:
@bliemli does this also work for the openshift flavored training? or is there an other naming scheme for the quota lab?
I wan't to explicit show the `-quota` because in the last two trainings this always was forgotten by the participants, despite its written in a note at top.

bliemli reviewed (commented):
Well, this is not a really nice solution. `<namespace>-quota` with a namespace named `xy-quota` would mean a user has to use `xy-quota-quota` which doesn't make any sense.

I'd prefer we put a big warning at the beginning of the lab.

splattner commented:
I changed the note to a warning, lets see if people read it. And also added the `--namespace <namespace>` to all commands to be consistent with the other lab chapters.
